### PR TITLE
Create IFP schema for HOTH API

### PR DESCRIPTION
**Pull request by juliusfitzhugh-ccpo**
- Create `Incremental Funding Plan` schema for generating IFP document in the HOTH API
- Update `GenerateDocumentRequest`

Ticket: AT-8379

### CODE REVIEW

File: hoth_api.yaml [hunk 1]
```diff
@@ -440,28 +442,52 @@ components:
           type: array
           items:
             type: string
+    IncrementalFundingPlan:
+      description: >-
+        Provides an Incremental Funding Plant (IFP) for an Acquisition Package
+      type: object
+      properties:
+        # ? is this the CSS Tracking Number/PostAwardId?
+        # DITCO will fill in doing review, will be left as TBD
```
Comment (jasonburkert-ccpo) [inline hoth_api.yaml:451]:
```suggestion
        # DITCO will fill in during review, will be left as TBD
```

File: hoth_api.yaml [hunk 2]
```diff
@@ -513,6 +539,36 @@ components:
             months and always rounded up to a whole number. Average weeks in a month 4.345.
             Average days in a month 30.4167. Example 6 wks / 4.345 = 1.381 => 2 months.
           type: integer
+    FundingDocument:
+      description: >-
+        Numbers from funding document(s). Either both the GT&C number and Order
+        number from US Treasury Bureau of the Fiscal Service Forms (FS_FORM)
```
Comment (jasonburkert-ccpo) [inline hoth_api.yaml:545]:
```suggestion
        number from US Treasury's Bureau of the Fiscal Service forms (FS_FORM)
```

File: hoth_api.yaml [hunk 3]
```diff
@@ -400,14 +400,16 @@ components:
...
         templatePayload:
           type: object
           oneOf:
             - $ref: "#/components/schemas/EvalPlan"
             - $ref: "#/components/schemas/DescriptionOfWork"
             - $ref: "#/components/schemas/IndependentGovernmentCostEstimate"
+            - $ref: "#/components/schemas/IncrementalFundingPlan"
     EvalPlan:
-      description: Provides the basis of evaluation for an Aquisition Package
+      description: Provides the basis of evaluation for an Acquisition Package
```
Comment (jasonburkert-ccpo) [inline hoth_api.yaml:412]: Good eye

**Review by jasonburkert-ccpo — CHANGES_REQUESTED** [review]

File: hoth_api.yaml [hunk 4]
```diff
@@ -440,28 +442,52 @@ components:
           type: array
           items:
             type: string
+    IncrementalFundingPlan:
+      description: >-
+        Provides an Incremental Funding Plant (IFP) for an Acquisition Package
+      type: object
+      properties:
+        # ? is this the CSS Tracking Number/PostAwardId?
```
Comment (jasonburkert-ccpo) [inline hoth_api.yaml:450]:
```suggestion
```

File: hoth_api.yaml (hunk 3 above)
Comment (juliusfitzhugh-ccpo) [reply hoth_api.yaml:412]: spell check helped 😆

File: hoth_api.yaml [hunk 5]
```diff
@@ -440,28 +442,52 @@ components:
           type: array
           items:
             type: string
+    IncrementalFundingPlan:
+      description: >-
+        Provides an Incremental Funding Plant (IFP) for an Acquisition Package
```
[the remark this answers is not in the document]
Comment (juliusfitzhugh-ccpo) [reply hoth_api.yaml:447]: Thanks for the description. Not sure how `Plant` slipped in there 😄.

**Review by jasonburkert-ccpo — APPROVED** [review]
Looks good!

**Pull request merged**